src/js/controller/AppController.js

```diff
@@ -15,33 +15,43 @@ L.DNC.AppController = L.Class.extend({
         this.mapView = new L.DNC.MapView();
         this.dropzone = new L.DNC.DropZone( this.mapView._map, {} );
         this.layerlist = new L.DNC.LayerList( { layerContainerId: 'dropzone' } ).addTo( this.mapView._map );
-        this.menuBar = new L.DNC.MenuBar()
-            .addTo( document.getElementById('menu-bar') )
-            .addChild( new L.DNC.Menu( "Geoprocessing Tools", {} )
-                .addChild(new L.DNC.TurfOperation('buffer', {
-                    maxFeatures: 1,
-                    additionalArgs: 0.1
-                }))
-                .addChild(new L.DNC.TurfOperation('union', {
-                    minFeatures: 2,
-                    maxFeatures: 2
-                }))
-                .addChild(new L.DNC.TurfOperation('erase', {
-                    minFeatures: 2,
-                    maxFeatures: 2
-                }))
-            );
+        
+        this.menubar = new L.DNC.MenuBar( { id: 'menu-bar' } );
 
-        this.notification = new L.DNC.Notifications();
+        // new menu
+        this.menu = {
+            geo: new L.DNC.Menu('Geoprocessing', this.menubar, { 
+                items: ['buffer', 'union', 'erase', 'intersect'] 
```

Comment:
If we're to merge this PR, we should probably either trim this list of the un-implemented `erase` and `intersect` or build out the tooling in `L.DNC.Geo` (I vote for the latter).

Interesting observation: if a menu is not defined, it will load the last options menu of the last operation clicked.

![bug](https://cloud.githubusercontent.com/assets/897290/7558736/5c84c69e-f768-11e4-9728-4c6339a65b78.gif)